### DISCUSSION TITLE
General Discussion Board

### DISCUSSION THREAD

**Discussion by iblowmymind** [devhawala/dwarf#9] [General]
First entry!

**Comment by iblowmymind** [comment]
Do you believe I should inquire Yasuhiro Hasegawa about the availability of the Clearinghouses still running?

**Reply by devhawala** [reply]
why not, but what for?

**Reply by iblowmymind** [reply]
I want to know if stuff like KSPA is still running to this day. A personal curiosity..
But I'm also asking if if it's running because if it is so, then there is a slight chance we can get the Xerox folks to release the software to the public.

**Comment by Kirtai** [comment]
This is very cool. I've wanted to play with Star/Viewpoint/Globalview for a long time.

**Comment by iblowmymind** [comment]
Okay, this is huge-
I just stumbled across a comment on a video about the 6085 that mentions an analyst (like the ones that's mentioned in Darkstar) software activation password "aaaaaaaaaah58" (that's 10 a's). It works with VP 2.x on any date (works with the clock set to 2024), I am yet to test VP 3.x but I think it would work there as well.

**Reply by iblowmymind** [reply]
It appears it doesn't work with VP 3.x.
